fastlane/lib/fastlane/setup/crashlytics_beta.rb

```diff
@@ -0,0 +1,89 @@
+module Fastlane
+  class CrashlyticsBeta
+    def run
+      UI.user_error!('Beta by Crashlytics configuration is currently only available for iOS projects.') unless Setup.new.is_ios?
+      config = {}
+      FastlaneCore::Project.detect_projects(config)
+      project = FastlaneCore::Project.new(config)
+      keys = keys_from_project(project)
+
+      if FastlaneFolder.setup?
+        UI.header('Copy and paste the following lane into your Fastfile to use Crashlytics Beta!')
+        puts lane_template(keys[:api_key], keys[:build_secret], project.schemes.first).cyan
+      else
+        fastfile = fastfile_template(keys[:api_key], keys[:build_secret], project.schemes.first)
+        FileUtils.mkdir_p('fastlane')
+        File.write('fastlane/Fastfile', fastfile)
```

Comment:
Yeah definitely, @mpirri and I had discussed this and we definitely plan on doing that.